fastlane/lib/fastlane/notification/slack.rb

```diff
@@ -0,0 +1,56 @@
+module Fastlane
+  module Notification
+    class Slack
+      def initialize(webhook_url)
+        @webhook_url = webhook_url
+        @client = Faraday.new do |conn|
+          conn.use(Faraday::Response::RaiseError)
+        end
+      end
+
+      # Overriding channel, icon_url and username is only supported in legacy incoming webhook.
+      # And use of attachments is now deprecated.
```

Comment:
```suggestion
      # Also note that the use of attachments has been deprecated by Slack, in favor of Block Kit.
```

Reply:
```suggestion
      # Also note that the use of attachments has been discouraged by Slack, in favor of Block Kit.
```

I rephrased it a little bit to be accurate😅